iina/PlayerCore.swift

```diff
@@ -447,9 +447,6 @@ class PlayerCore: NSObject {
     let a = mpv.getDouble(MPVOption.PlaybackControl.abLoopA)
     let b = mpv.getDouble(MPVOption.PlaybackControl.abLoopB)
     if a != 0 || b != 0 {
```

Comment:
Is it still necessary to check these values? Can we just reset `info.abLoopStatus` to 0 without checking?

Reply:
I'm looking for how to get the option string of IINA, because we do not know if user have overwritten the config value that whether to reset AB loop when opening next.

Reply:
So yes I don't think we need to check the ab loop time value, and it is necessary to check the option value

Reply:
That's a good point; if you want to cover all possible cases, it's possible that mpv loads ab loop status from watch_later files, or user controls ab loop via a lua script or a local socket.

So a perfect solution should listen to `ab-loop-a` and `ab-loop-b` changes and update corresponding status (probably send OSD as well), just like how IINA deals with a lot of other properties.

However I think there's very little chance for a user to override this option, so it's OK to update it only when file loaded.

I just found if you get `ab-loop-a` or `b` using `getFlag()`, it will return `false` when the loop point is not set, which I think is better than checking if it's zero; and then you can set `info.abLoopStatus` to 0, 1 or 2 depends on the two flags.